plugins/tracker-resources/src/components/issues/Active.svelte

```diff
@@ -1,10 +1,28 @@
+<!--
+// Copyright © 2022 Hardcore Engineering Inc.
+// 
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// 
+// See the License for the specific language governing permissions and
+// limitations under the License.
+-->
 <script lang="ts">
   import { Ref } from '@anticrm/core'
-  import { IssueStatus, Team } from '@anticrm/tracker'
-  import Issues from './Issues.svelte'
+  import { Team } from '@anticrm/tracker'
   import tracker from '../../plugin'
+  import Issues from './Issues.svelte'
 
   export let currentSpace: Ref<Team>
 </script>
 
-<Issues {currentSpace} categories={[IssueStatus.InProgress, IssueStatus.Todo]} title={tracker.string.ActiveIssues} />
+<Issues
+  {currentSpace}
+  statusCategories={new Set([tracker.issueStatusCategory.Unstarted, tracker.issueStatusCategory.Started])}
```

Comment:
Why we need a set here, it would be more nice to pass just an array and construct a set inside component if required.